### PR TITLE
Fixes #399

### LINKED ISSUE #399: Bug with ToTensorV2
## 🐛 Bug
Error with `ToTensorV2` (sorry for my previous faulty PR).

```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-53-ff0c00a2db0e> in <module>
     10 res = t(**kwargs)
     11 
---> 12 res2 = A.ReplayCompose.replay(res['replay'], **kwargs)

/opt/conda/lib/python3.7/site-packages/albumentations/core/composition.py in replay(saved_augmentations, **kwargs)
    295     @staticmethod
    296     def replay(saved_augmentations, **kwargs):
--> 297         augs = ReplayCompose._restore_for_replay(saved_augmentations)
    298         return augs(force_apply=True, **kwargs)
    299 

/opt/conda/lib/python3.7/site-packages/albumentations/core/composition.py in _restore_for_replay(transform_dict, lambda_transforms)
    322                 args["transforms"] = [
    323                     ReplayCompose._restore_for_replay(t, lambda_transforms=lambda_transforms)
--> 324                     for t in args["transforms"]
    325                 ]
    326             transform = cls(**args)

/opt/conda/lib/python3.7/site-packages/albumentations/core/composition.py in <listcomp>(.0)
    322                 args["transforms"] = [
    323                     ReplayCompose._restore_for_replay(t, lambda_transforms=lambda_transforms)
--> 324                     for t in args["transforms"]
    325                 ]
    326             transform = cls(**args)

/opt/conda/lib/python3.7/site-packages/albumentations/core/composition.py in _restore_for_replay(transform_dict, lambda_transforms)
    324                     for t in args["transforms"]
    325                 ]
--> 326             transform = cls(**args)
    327 
    328         transform.params = params

TypeError: __init__() got an unexpected keyword argument 'always_apply'
```

## To Reproduce
```python
import albumentations as A
from albumentations.pytorch import ToTensorV2


t = A.ReplayCompose([
    ToTensorV2()
])

kwargs = {
    'image': np.ones((20, 20, 3), dtype='uint8'),
    'mask': np.ones((20, 20, 3), dtype='uint8'),    
}

res = t(**kwargs)
res2 = A.ReplayCompose.replay(res['replay'], **kwargs)
```

## Environment

 - Albumentations version (e.g., 0.1.8): stable
 - Python version (e.g., 3.7): 3.7
 - OS (e.g., Linux): linux
 - How you installed albumentations (`conda`, `pip`, source): pip
 - Any other relevant information:

### PR DESCRIPTION
**Pull request by vfdev-5**
Fixes #399

- modified `__init__`
- added a test with `ReplayCompose`

Please review

### CODE REVIEW

**Comment by albu** [comment]
I think we shouldn't give ability to not apply totensor.. maybe fix it from the other side?

**Comment by Dipet** [comment]
We can use `**kwargs` without passing it to super constructor

**Comment by vfdev-5** [comment]
Shouldn't it behave similarly to `FromFloat` and similar ?

File: albumentations/pytorch/transforms.py [hunk 1]
```diff
@@ -83,8 +83,8 @@ class ToTensorV2(BasicTransform):
     """Convert image and mask to `torch.Tensor`.
     """
 
-    def __init__(self):
-        super(ToTensorV2, self).__init__(always_apply=True)
+    def __init__(self, always_apply=False, p=1.0):
```
Comment (albu) [inline albumentations/pytorch/transforms.py]:
```suggestion
    def __init__(self, always_apply=True, p=1.0):
```

**Comment by albu** [comment]
okay, let's make it `always_apply` by default, it should be fine

**Comment by vfdev-5** [comment]
2 approvals, let's merge it :)

**Pull request merged**